From c9a46ed4beee5688e2f0c44ddb36800876594139 Mon Sep 17 00:00:00 2001
From: Brandon Bennett <bennetb@gmail.com>
Date: Sat, 19 Nov 2011 16:02:04 -0700
Subject: [PATCH 08/45] Add ext2, ext3, and vfat to releasetools

Support some non-standard partition that may be found in recovery.fstab

Change-Id: I0026a1d3ae5d432db10150d1a15f7bc8e7a6054b
---
 tools/releasetools/common.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/tools/releasetools/common.py b/tools/releasetools/common.py
index 632c1e2..28b2e05 100644
--- a/tools/releasetools/common.py
+++ b/tools/releasetools/common.py
@@ -2077,7 +2077,10 @@ PARTITION_TYPES = {
     "ext4": "EMMC",
     "emmc": "EMMC",
     "f2fs": "EMMC",
-    "squashfs": "EMMC"
+    "squashfs": "EMMC",
+    "ext2": "EMMC",
+    "ext3": "EMMC",
+    "vfat": "EMMC"
 }
 
 
-- 
2.7.4

